src/components/sidebar/routePathView/RoutePathView.tsx

```diff
@@ -24,6 +25,7 @@ interface IRoutePathViewState {
     hasModifications: boolean;
     routePath: IRoutePath | null;
     isLoading: boolean;
+    invalidFieldsDictionary: object;
```

Comment:
I think it is more common just to use "map" rather than "dictionary". If that is what the word dictionary is implying here.

Reply:
Yes, you are correct, map is probably a better name

Reply:
This is now fixed